pkg/watcher/resource-event-handler/pod.go

```diff
@@ -56,18 +56,17 @@ func (p *podEventHandler) OnAdd(obj interface{}) {
 	}
 
 	if err := p.addNetworksFromPod(pod); err != nil {
-		err = fmt.Errorf("OnAdd(): %v", err)
-		glog.Error(err)
+		log.Err(err)
 		return
 	}
 
-	glog.V(3).Infof("OnAdd(): successfully added namespace %s name %s", pod.Namespace, pod.Name)
+	log.Info().Msgf("OnAdd(): successfully added namespace %s name %s", pod.Namespace, pod.Name)
 }
 
 func (p *podEventHandler) OnUpdate(oldObj, newObj interface{}) {
-	glog.V(4).Infof("OnUpdate(): oldPod %v, newPod %v", oldObj, newObj)
+	log.Debug().Msgf("OnUpdate(): oldPod %v, newPod %v", oldObj, newObj)
 	pod := newObj.(*kapi.Pod)
-	glog.Infof("OnUpdate(): namespace %s name %s", pod.Namespace, pod.Name)
+	log.Info().Msgf("OnUpdate(): namespace %s name %s", pod.Namespace, pod.Name)
 
 	if !utils.PodWantsNetwork(pod) {
```

Comment:
same as my comment above about possibly adding logs to the if statements if relevant